Reviewing docs for errors and mistakes uptill concepts section (#53)
openmrs/openmrs-contrib-rest-api-docs: source/includes/Encounter/encounter.md

```console
  DELETE /encounter/:target_encounter_uuid?purge=true
```

> ## List encounter provider subresources

Reviewer: ```suggestion
## List encounter provider subresources
```
```suggestion
## List encounter provider sub-resources
```

> * ### List all encounter provider subresources for a visit.

Reviewer: ```suggestion
* ### List all encounter provider sub-resources for a visit.
```

GET /encounter/:target_encounter_uuid/encounterprovider 
```

> * ### List encounter provider subresources by it's UUID and parent encounter UUID.

Reviewer: ```suggestion
* ### List encounter provider sub-resources by it's UUID and parent encounter UUID.
```

```
## Create an encounter provider sub resource with properties

> * To Create an attribute subresource for a specific visit resource, you need to specify below attributes in the request body.

Reviewer: ```suggestion
* To create an attribute sub-resource for a specific visit resource, you need to specify below attributes in the request body.
```

```
 
 

> ## Update encounter provider subresource

Reviewer: ```suggestion
## Update encounter provider sub-resource
```

  "encounterRole": "240b26f9-dd88-4172-823d-4a8bfeb7841f"
}
```

> ## Delete encounter provider subresource

Reviewer: ```suggestion
## Delete encounter provider sub-resource
```

> * Delete or Voided a target encounter provider subresource by its UUID. Returns a `404 Not Found` status if attribute not exists. 

Reviewer: ```suggestion
* Delete or Voided a target encounter provider sub-resource by its UUID. Returns a `404 Not Found` status if attribute not exists. 
```

Reviewer 2: @sanjulamadurapperuma In the docs in other sections as well subresource is used uniformly all over thats why I have adhered to it here as well.

Reviewer: @LuGO0 It should actually be _sub-resources_ in any context. We have used _sub-resources_ in our [Wiki](https://wiki.openmrs.org/display/docs/Documenting+REST+Resources) as well. Could you point to some other doc sections using _subresources_?
